spec/models/pro_spec.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+RSpec.describe Pro do
+  describe '.site' do
+    it 'should default to "http://pro.europeana.eu"' do
+      expect(described_class.site).to eq('http://pro.europeana.eu')
```

Comment:
I don't think that would work as by the time the RSpec examples are run, that call to ENV has already occurred. I have reworked the specs to not fail when EUROPEANA_PRO_URL is set in the test env.